docs/guides/cloud/data-storage-and-masking.mdx

```diff
@@ -0,0 +1,168 @@
+
+- The rendered DOM and CSS styles for the application under test
+- Cypress commands and events represented in the Command Log
+- Network traffic within your application under test
+- Browser console logs
+
+### Screenshot controls
+
+#### Blackout elements in screenshots
+
+Cypress offers an option to black out (mask) an array of elements from screenshots. [See screenshot API docs.](/api/commands/screenshot#Arguments) This allows you to black out specific elements on screenshots so that they will not be visible in the locally saved screenshots or to any users that have access to the run in Cypress Cloud.
+
+Add a control to your [support file](/guides/references/configuration#Testing-Type-Specific-Options) to set the default blackout options for all screenshots.
+
+```js
+before(() => {
+  Cypress.Screenshot.defaults({
+    // blackout all elements with 'mask-cy' attribute
+    blackout: ['mask-cy'],
+  })
+})
+```
+
+Add a control for a specific screenshot to blackout specific elements.
+
+```js
+it('admin page loads', () => {
+  cy.visit('/admin')
+  cy.screenshot({
+    blackout: ['.user-name'],
+  })
+})
+```
+
+#### Remove Cypress Command Log from screenshots
+
+Cypress offers an option to hide the Cypress Command Log from screenshots so that no information from that area will be captured in screenshots. [See `cy.screenshot` docs](/api/commands/screenshot#Arguments).
+
+Add a control to your [support file](/guides/references/configuration#Testing-Type-Specific-Options) to set the default capture options for all screenshots.
+
+```js
+before(() => {
+  Cypress.Screenshot.defaults({
+    capture: 'viewport', // or fullPage
+  })
+})
+```
+
+Add a control for a specific screenshot to only capture the application under test.
+
+```js
+it('sign in page', () => {
+  cy.visit('/sign-in')
+  cy.screenshot({
+    capture: 'viewport', // or fullPage
+  })
+})
+```
+
+### Video controls
+
+#### Disable video upload
+
+You can delete videos that are recorded when you are passing `video: true` by following [this guide to control what is uploaded to Cypress Cloud](/guides/guides/screenshots-and-videos#Control-which-videos-to-keep-and-upload-to-Cypress-Cloud).
+
+### Cypress Command Log controls
+
+#### Turn off Cypress Command Log rendering
+
+You can disable showing the entirety of the Cypress Command Log as well as the Application under Test (AUT) controls in your tests to prevent all test content from being shown and recorded in any screenshot or video by passing `--no-runner-ui` via the CLI while running Cypress tests. [See docs.](/guides/guides/command-line#Options). This means that the AUT will be the only thing that is rendered while running tests, so the only thing that is captured in screenshots and videos.
```

Comment:
Ended up with a double period here. Also one more more descriptive text link. 

```suggestion
You can disable showing the entirety of the Cypress Command Log as well as the Application under Test (AUT) controls in your tests to prevent all test content from being shown and recorded in any screenshot or video by passing `--no-runner-ui` via the CLI while running Cypress tests [See Cypress Command Line options.](/guides/guides/command-line#Options). This means that the AUT will be the only thing that is rendered while running tests, so the only thing that is captured in screenshots and videos.
```